### PR TITLE
[WIP] Add Intake hardware support & control commands

### PR DESCRIPTION
**Pull request by rsninja722**

### CODE REVIEW

**Comment by ewpratten** [comment]
CI Failed with error:

```
 /github/workspace/src/main/java/frc/robot/subsystems/cellmech/Hopper.java:350: error: method getCellCount() is already defined in class Hopper
    public int getCellCount() {
```

File: src/main/java/frc/robot/autonomous/actions/subsystemActions/Intake.java
Comment (ewpratten) [inline src/main/java/frc/robot/autonomous/actions/subsystemActions/Intake.java]: target should be equal to cellCount

File: src/main/java/frc/robot/autonomous/actions/subsystemActions/Intake.java
Comment (ewpratten) [inline src/main/java/frc/robot/autonomous/actions/subsystemActions/Intake.java]: Please name this something useful

File: src/main/java/frc/robot/autonomous/actions/subsystemActions/Intake.java
Comment (ewpratten) [inline src/main/java/frc/robot/autonomous/actions/subsystemActions/Intake.java]: This also needs a useful name

File: src/main/java/frc/robot/subsystems/cellmech/Hopper.java
Comment (ewpratten) [inline src/main/java/frc/robot/subsystems/cellmech/Hopper.java]: Holding a ball at any of these sensors will cause the ball count to rapidly increase or drop. These should be latched

File: src/main/java/frc/robot/subsystems/cellmech/Intake.java
Comment (ewpratten) [inline src/main/java/frc/robot/subsystems/cellmech/Intake.java]: This is apparently 2 talons following eachother, not one

**Review by ewpratten — CHANGES_REQUESTED** [review]

**Comment by rsninja722** [comment]
Made the action purely a stub, reverted changes to superstructure, made roller use 2 motors. I fail to see how the cell count code will rapidly change, it only changes the cell count when the sensor changes states, and now only if the belt is supposed to be moving.

**Comment by rsninja722** [comment]
I will make superstructure have actual function on Tuesday

**Comment by ewpratten** [comment]
So you un-did your work? I cant check the diff right now

**Comment by ewpratten** [comment]
Requesting @awpratten for review

**Comment by rsninja722** [comment]
This was supposed to be just getting the hopper and intake into master, but then Will asked for a stub of an intake command for making autos. I don't know why I tried to make the stub have any function in the first place.

**Comment by ewpratten** [comment]
ah, ok

**Review by ewpratten — APPROVED** [review]

**Comment by ewpratten** [comment]
This PR is waiting on IRL testing, and #126 before merge

**Comment by ewpratten** [comment]
Replaced in favour of #146

**Pull request closed without merging**
